semgrep/semgrep/job_postings.py

```diff
@@ -0,0 +1,48 @@
+from typing import Dict
+from typing import List
+
+import requests
+
+from semgrep.constants import RETURNTOCORP_LEVER_URL
+
+
+def print_job_postings() -> None:
+    # http get postings
+    response = requests.get(RETURNTOCORP_LEVER_URL, timeout=30)
+    postings = response.json()
+
+    # organise per team
+    postings_dict: Dict[str, List[Dict[str, str]]] = {}
+    for posting in postings:
+        team = posting["categories"]["team"]
+        if team not in postings_dict:
+            postings_dict[team] = []
+
+        title = posting["text"]
+        url = posting["hostedUrl"]
+        postings_dict[team].append({"title": title, "url": url})
+
+    print(
+        "We love that you're using Semgrep. Would you like to... work on it full time?! 😄 "
+    )
+    print("")
+    # iterate over teams alphabetically
+    for team in sorted(postings_dict):
+        team_character_count = len(team)
+
+        print_n_hashtags(team_character_count + 4)
```

Comment:
```suggestion
        print((team_character_count + 4) * "#")
```

I think the separate function is not adding much value over writing this.